tests/admin_widgets/test_autocomplete_widget.py

```diff
@@ -54,7 +54,7 @@ def test_build_attrs(self):
             'data-ajax--cache': 'true',
             'data-ajax--delay': 250,
             'data-ajax--type': 'GET',
-            'data-ajax--url': '/admin_widgets/band/autocomplete/',
+            'data-ajax--url': '/admin_widgets/band/autocomplete/?to_field=id',
```

Comment:
Again, the attribute shouldn't be part of the URL, but a separate `data`-attribute.